### DISCUSSION TITLE
How to check who starred your repos?

### DISCUSSION THREAD

**Discussion by ghost** [community/community#18466] [Other Feature Feedback, Questions, & Ideas] [answered]
Can't seem to find this feature now... can anyone help?

**Comment by melvincwng** [comment, marked as answer]
Add '/stargazers' to the end of the URL of your code repository.

E.g. https://github.com/roshanMB/roshanMB/stargazers

Hope this clarifies.

**Comment by number0x0001** [comment]
Hello! It's extremely simple - all you have to do is add **/stargazers** at the end of your repository's URL.
